modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-fido2-web/src/main/resources/META-INF/resources/mfa_fido2_checker/verify_browser.jsp

```diff
@@ -0,0 +1,82 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ *
+ *
+ *
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+String assertionReqest = (String)request.getAttribute("assertionReqest");
+%>
+
+<div id="<portlet:namespace/>messageContainer"></div>
+
+<aui:button id="startAuthentication" value="start" />
+
+<aui:input name="responseJSON" showRequiredLabel="yes" type="hidden" />
+
+<aui:button-row>
+	<aui:button type="submit" value="submit" />
```

Comment:
Same for this button in verification.

Reply:
@arthurchan35, since I can't test the verification step because I don't have a secure key could you please test and confirm this is working as expect when you have a change, thanks!